test/functional/action-params.spec.ts

```diff
@@ -72,6 +78,46 @@ describe("action parameters", () => {
         // reset metadata args storage
         defaultMetadataArgsStorage().reset();
 
+
+        @Middleware()
+        class SessionMiddleware implements MiddlewareInterface {
```

Comment:
I guess, this is because the `Middleware` in the `global-options` is imported with `options`.

https://github.com/pleerock/routing-controllers/blob/master/test/functional/load-from-directory.spec.ts#L72